esp32_s3_lcd_ev_board/sub_board/sub_board_480x480.c

```diff
@@ -0,0 +1,374 @@
+    {0x68, {0x00, 0x08, 0x15, 0x08, 0x15, 0x7a, 0x7a, 0x08, 0x15, 0x08, 0x15, 0x7a, 0x7a}, 13},
+    {0x60, {0x38, 0x08, 0x7a, 0x7a, 0x38, 0x09, 0x7a, 0x7a}, 8},
+    {0x63, {0x31, 0xe4, 0x7a, 0x7a, 0x31, 0xe5, 0x7a, 0x7a}, 8},
+    {0x69, {0x04, 0x22, 0x14, 0x22, 0x14, 0x22, 0x08}, 7},
+    {0x6b, {0x07}, 1},
+    {0x7a, {0x08, 0x13}, 2},
+    {0x7b, {0x08, 0x13}, 2},
+    {0xd1, {0x00, 0x00, 0x00, 0x04, 0x00, 0x12, 0x00, 0x18, 0x00, 0x21, 0x00, 0x2a, 0x00, 0x35, 0x00, 0x47, 0x00,
+            0x56, 0x00, 0x90, 0x00, 0xe5, 0x01, 0x68, 0x01, 0xd5, 0x01, 0xd7, 0x02, 0x36, 0x02, 0xa6, 0x02, 0xee,
+            0x03, 0x48, 0x03, 0xa0, 0x03, 0xba, 0x03, 0xc5, 0x03, 0xd0, 0x03, 0xe0, 0x03, 0xea, 0x03, 0xfa, 0x03,
+            0xff}, 52},
+    {0xd2, {0x00, 0x00, 0x00, 0x04, 0x00, 0x12, 0x00, 0x18, 0x00, 0x21, 0x00, 0x2a, 0x00, 0x35, 0x00, 0x47, 0x00,
+            0x56, 0x00, 0x90, 0x00, 0xe5, 0x01, 0x68, 0x01, 0xd5, 0x01, 0xd7, 0x02, 0x36, 0x02, 0xa6, 0x02, 0xee,
+            0x03, 0x48, 0x03, 0xa0, 0x03, 0xba, 0x03, 0xc5, 0x03, 0xd0, 0x03, 0xe0, 0x03, 0xea, 0x03, 0xfa, 0x03,
+            0xff}, 52},
+    {0xd3, {0x00, 0x00, 0x00, 0x04, 0x00, 0x12, 0x00, 0x18, 0x00, 0x21, 0x00, 0x2a, 0x00, 0x35, 0x00, 0x47, 0x00,
+            0x56, 0x00, 0x90, 0x00, 0xe5, 0x01, 0x68, 0x01, 0xd5, 0x01, 0xd7, 0x02, 0x36, 0x02, 0xa6, 0x02, 0xee,
+            0x03, 0x48, 0x03, 0xa0, 0x03, 0xba, 0x03, 0xc5, 0x03, 0xd0, 0x03, 0xe0, 0x03, 0xea, 0x03, 0xfa, 0x03,
+            0xff}, 52},
+    {0xd4, {0x00, 0x00, 0x00, 0x04, 0x00, 0x12, 0x00, 0x18, 0x00, 0x21, 0x00, 0x2a, 0x00, 0x35, 0x00, 0x47, 0x00,
+            0x56, 0x00, 0x90, 0x00, 0xe5, 0x01, 0x68, 0x01, 0xd5, 0x01, 0xd7, 0x02, 0x36, 0x02, 0xa6, 0x02, 0xee,
+            0x03, 0x48, 0x03, 0xa0, 0x03, 0xba, 0x03, 0xc5, 0x03, 0xd0, 0x03, 0xe0, 0x03, 0xea, 0x03, 0xfa, 0x03,
+            0xff}, 52},
+    {0xd5, {0x00, 0x00, 0x00, 0x04, 0x00, 0x12, 0x00, 0x18, 0x00, 0x21, 0x00, 0x2a, 0x00, 0x35, 0x00, 0x47, 0x00,
+            0x56, 0x00, 0x90, 0x00, 0xe5, 0x01, 0x68, 0x01, 0xd5, 0x01, 0xd7, 0x02, 0x36, 0x02, 0xa6, 0x02, 0xee,
+            0x03, 0x48, 0x03, 0xa0, 0x03, 0xba, 0x03, 0xc5, 0x03, 0xd0, 0x03, 0xe0, 0x03, 0xea, 0x03, 0xfa, 0x03,
+            0xff}, 52},
+    {0xd6, {0x00, 0x00, 0x00, 0x04, 0x00, 0x12, 0x00, 0x18, 0x00, 0x21, 0x00, 0x2a, 0x00, 0x35, 0x00, 0x47, 0x00,
+            0x56, 0x00, 0x90, 0x00, 0xe5, 0x01, 0x68, 0x01, 0xd5, 0x01, 0xd7, 0x02, 0x36, 0x02, 0xa6, 0x02, 0xee,
+            0x03, 0x48, 0x03, 0xa0, 0x03, 0xba, 0x03, 0xc5, 0x03, 0xd0, 0x03, 0xe0, 0x03, 0xea, 0x03, 0xfa, 0x03,
+            0xff}, 52},
+    {0x3a, {0x66}, 1},
+    {0x11, {0x00}, 0},
+    {0x00, {0x00}, 0xff},
+};
+// *INDENT-OFF*
+
+/**
+ * @brief Configure LCD with specific commands and data
+ *
+ * @param[in] io_expander: IO expander handle
+ *
+ * @return
+ *      - ESP_OK: Success, otherwise returns ESP_ERR_xxx
+ *
+ */
+static esp_err_t lcd_config(esp_io_expander_handle_t io_expander)
```

Comment:
Yeah, it also confused me at first. This LCD type is called "3 Wire SPI + Parallel RGB Interface". You can read this [article](https://focuslcds.com/3-wire-spi-parallel-rgb-interface-fan4213/) to understand it.

Reply:
I am thinking about this controller (GC9503CV) and is it possible make it as separate esp_lcd component? It could be, if the SPI lines wouldn't on IO expander. But I don't know, if we can do it in this case... What do you think?

Reply:
Yeah, we can make it as separate esp_lcd component. But as you say, the SPI lines are fixed on IO expander. Actually, this board is limited by a lack of IO as it needs to have I2S, I2C, USB, and Octal SPSRAM at the same time. Fortunately, the SPI function of the LCD is only used for register configuration, and the transmission rate is not high, so it can be simulated by the pin of the IO expander.

Reply:
What's more, the `esp_lcd` driver in IDF doesn't support 3-wire (also called 9-bit) SPI now. You can take a look [here](https://github.com/espressif/esp-idf/blob/release/v5.0/components/esp_lcd/src/esp_lcd_panel_io_spi.c#L69-L71).

Reply:
Ok, I understand. Thank you for info.

Reply:
Is this resolved @espzav ?